deploy/kube-bench/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-bench-ds
```

Comment:
wouldn't it be better to use the chart/release name here (`kube-bench.fullname` template)? (example in the generated templates when you run `helm create`)